core/src/main/java/com/linecorp/armeria/client/logging/LoggingClientBuilder.java

```diff
@@ -103,14 +122,7 @@ public LoggingClientBuilder failureResponseLogLevel(LogLevel failedResponseLogLe
     }
 
     @Override
-    public LoggingClientBuilder requestLogLevelMapper(
-            Function<? super RequestOnlyLog, LogLevel> requestLogLevelMapper) {
-        return (LoggingClientBuilder) super.requestLogLevelMapper(requestLogLevelMapper);
-    }
-
-    @Override
-    public LoggingClientBuilder responseLogLevelMapper(
-            Function<? super RequestLog, LogLevel> responseLogLevelMapper) {
```

Comment:
These look breaking changes. Could we revive the old methods and mark `@Deprecated`?